src/UI/App.jsx

```diff
@@ -37,6 +37,7 @@ export default class App extends React.Component {
 		const interfaceTextsDefaults = interfaceLanguage === "nl" ? InterfaceTexts.dutch : InterfaceTexts.english;
 		this.state = {
 			showChat: false,
+			messengerOpenState: localStorage.getItem("messengerOpenState"),
```

Comment:
als de key niet bestaat is dit `null` maar dat is geen state?
Misschien als default `minimze` aangezien `showChat` als default `false` heeft

Reply:
misschien beter:

```jsx
const showChat = false
this.state = {
			showChat,
			messengerOpenState: showChat ? 'open' : 'minimze'
```

Reply:
Ik heb jou suggestie voor de state toegepast, momenteel kijken wij rechtstreeks naar de storage voor veranderingen, maar voor het css zou ik gebruik kunnen maken van deze state. Zie: c98499f68cd8a2736b81d52d55b71864cb8dda42

Reply:
Resolved